gm4_furniture/data/gm4_furniture/functions/place/furniture/furniture_station.mcfunction

```diff
@@ -0,0 +1,48 @@
+# try to place the furniture_station
+# @s = player who placed the furniture player head
+# at the center of the placed block rotated along axis to face towards player or away from the wall
+# run from place/resolve_id
+
+# first perform checks to see if furniture fits where it was placed
+scoreboard players set $valid_placement gm4_furniture_data 1
+execute unless block ~ ~1 ~ #gm4:replaceable run scoreboard players set $valid_placement gm4_furniture_data 0
+execute positioned ~ ~1 ~ align y if entity @e[type=interaction,tag=gm4_furniture,tag=!gm4_new_furniture,distance=..0.1] run scoreboard players set $valid_placement gm4_furniture_data 0
+
+execute if score $placement_blocked gm4_furniture_data matches 1 run return 0
+
+# if placement is not valid cancel placement
+execute if score $valid_placement gm4_furniture_data matches 0 run loot spawn ~ ~ ~ loot gm4_furniture:items/furniture_station
+execute if score $valid_placement gm4_furniture_data matches 0 run setblock ~ ~ ~ air
+execute if score $valid_placement gm4_furniture_data matches 0 run return 0
+
+# spawn the furniture_station
+summon item_display ~ ~-0.4999 ~ {Tags:["gm4_furniture","gm4_furniture.display","gm4_furniture.furniture_station","smithed.entity","smithed.strict","gm4_new_furniture"],CustomName:'"gm4_furniture_display.furniture_station"',item:{id:"leather_horse_armor",Count:1,tag:{data:{furniture_id:"furniture_station"},CustomModelData:3420150}},item_display:head,Rotation:[0.0f,0.0f],transformation:{left_rotation:[0f,0f,0f,1f],right_rotation:[0f,0f,0f,1f],translation:[0f,0.5f,0f],scale:[1f,1f,1f]}}
```

Comment:
It uses 1 interaction for the main furniture, then 3 more for the different tools. It's done that way since the 3 tool displays will swap between certain displayed tools.

There might be some way to cut down on the amount, but that would require some thought as to how to work the resource pack

Reply:
I would recommend either that and/or a slow clock that checks for nearby players and reduces the entire furniture station to 1 marker when not in range.